src/DonorProfiles/Repositories/Donations.php

```diff
@@ -194,18 +194,24 @@ protected function getFormInfo( $payment ) {
 	 */
 	protected function getPaymentInfo( $payment ) {
 
+		$pdfReceiptUrl = '';
+		if ( class_exists( 'Give_PDF_Receipts' ) ) {
+			$pdfReceiptUrl = esc_url( \Give_PDF_Receipts::get_instance()->engine->get_pdf_receipt_url( $payment->id ) );
```

Comment:
Looking at the definition of `get_pdf_receipt_url`, this value is already being escaped as a URL before being returned.

Reply:
This issue was addressed by @kjohnson, ref: https://github.com/impress-org/givewp/pull/5613/commits/a7310179780b8971d6541cfcac3540fb4a01cb87